client/src/pages/models/train/[name].tsx

```diff
@@ -0,0 +1,22 @@
+import Prose from 'components/Prose';
+import {useRouter} from 'next/router';
+import React from 'react';
+
+export default function ModelTrainingView() {
+  const router = useRouter();
+  const {name} = router.query;
```

Comment:
Not 100% sure on this but, if this is the name of the model, would it be better to call it `modelName`?